modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -23,6 +23,7 @@ In {product-title} 4.13, the `stable` channel has been removed. Before upgrading
 |*OpenShift GitOps* 7+|*Component Versions*|*OpenShift Versions*
 
 |*Version* |*`kam`*    |*Helm*  |*Kustomize* |*Argo CD*|*ApplicationSet* |*Dex*     |*RH SSO* |
+|1.9.0    |0.0.48 TP |3.11.2 GA|5.0.1 GA   |2.7.2 GA |NA     |2.35.1 GA |7.5.1 GA |4.12-4.13
```

Comment:
@wtam2018 Shouldn't KAM version be 0.0.49?

Reply:
It should be 0.0.49 as that is the version with the golang upgrade. Just updated it in the component matrix as it wasn't updated there and that's probably why it wasn't updated here.

Reply:
@varshab1210 and @reginapizza: Addressed in the revised version now.